packages/element-lib/src/adapters/database/ElementCouchDBAdapter.spec.js

```diff
@@ -0,0 +1,113 @@
+const ElementCouchDBAdapter = require('./ElementCouchDBAdapter');
+
+// Needs a prod cloudant URL to run
+describe.skip('ElementCouchDBAdapter', () => {
```

Comment:
option we could run couchdb locally in travis and have integration tests for this use case in the api.

Reply:
Created an issue for that: https://github.com/decentralized-identity/element/issues/109